### PR TITLE
Added blog post announcing postgres support

### PR DESCRIPTION
**Pull request by tempoz**
**Related issues**: N/A

### CODE REVIEW

File: website/blog/postgres-support.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+slug: postgres-support
+title: Postgres Support for BuildBuddy
+description: We're happy to announce postgres support for BuildBuddy.
+author: Zoey Greer
+author_title: Engineer @ BuildBuddy
+date: 2023-06-09:12:00:00
+author_url: https://www.github.com/tempoz
+author_image_url: https://avatars.githubusercontent.com/u/735684?v=4
+tags: [product]
```
[the remark this answers is not in the document]
Comment (siggisim) [reply website/blog/postgres-support.md:10]: postgres

**Review by tylerwilliams — APPROVED** [review]
LGTM but please also get @siggisim's approval / see if he has a date in mind :)

**Review by siggisim — APPROVED** [review]
lgtm, let's make the release date in the header tomorrow (you can merge whenever) and i'll make noise about it then

File: website/blog/postgres-support.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+slug: postgres-support
+title: PostgreSQL Support for BuildBuddy
+description: We're happy to announce PostgreSQL support for BuildBuddy.
+author: Zoey Greer
+author_title: Engineer @ BuildBuddy
+date: 2023-06-15:12:00:00
```
Comment (tempoz) [inline website/blog/postgres-support.md:7]:
```suggestion
date: 2023-06-21:12:00:00
```

**Pull request merged**